students/minseok/westagram/users/views.py

```diff
@@ -27,11 +28,13 @@ def post(self, request):
             
             if User.objects.filter(email=email).exists():
                 return JsonResponse({"message" : "The email already exists"}, status=400)
+            
+            hashed_pw = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Comment:
변수명은 최대한 줄이지 말고 사용해 주세요.
```suggestion
            hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```